Lombiq.VueJs.Samples/Assets/Scripts/VueComponents/qr-card.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <div class="container">
+        <div class="row justify-content-center mb-3">
+            <div class="col-md-6 col-sm">
+                <div class="card">
+                    <div class="card-body">
+                        <h5 v-if="!loading && !error">[[ Scan code ]]</h5>
+                        <qrcode-stream @decode="onDecode" @init="onInit" :track="paintOutline">
+                            <loading-indicator v-if="loading">
+                                [[ Loading ... ]]
+                            </loading-indicator>
+                        </qrcode-stream>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="row justify-content-center mb-3" v-if="error">
+            <div class="col-md-6 col-sm">
+                <p v-if="error === 'NotAllowedError'" class="error">[[ You need to grant camera access permission. ]]</p>
+                <p v-else-if="error === 'NotFoundError'" class="error">[[ No camera found. ]]</p>
+                <p v-else-if="error === 'NotSupportedError'" class="error">[[ Secure context required (HTTPS, localhost). ]]</p>
+                <p v-else-if="error === 'NotReadableError'" class="error">[[ The camera is already in use. ]]</p>
+                <p v-else-if="error === 'OverconstrainedError'" class="error">[[ Installed cameras are not suitable. ]]</p>
+                <p v-else-if="error === 'StreamApiNotSupportedError'" class="error">[[ The stream API is not supported in this browser. ]]</p>
+                <p v-else-if="error === 'InsecureContextError'" class="error">[[ Camera access is only permitted in secure context. Use HTTPS or localhost rather than HTTP. ]]</p>
+                <p v-else class="error">[[ Camera error. ]]</p>
```

Comment:
Let's use some working error classes:

```suggestion
                <p v-if="error === 'NotAllowedError'" class="message-error px-1">[[ You need to grant camera access permission. ]]</p>
                <p v-else-if="error === 'NotFoundError'" class="message-error px-1">[[ No camera found. ]]</p>
                <p v-else-if="error === 'NotSupportedError'" class="message-error px-1">[[ Secure context required (HTTPS, localhost). ]]</p>
                <p v-else-if="error === 'NotReadableError'" class="message-error px-1">[[ The camera is already in use. ]]</p>
                <p v-else-if="error === 'OverconstrainedError'" class="message-error px-1">[[ Installed cameras are not suitable. ]]</p>
                <p v-else-if="error === 'StreamApiNotSupportedError'" class="message-error px-1">
                    [[ The stream API is not supported in this browser. ]]
                </p>
                <p v-else-if="error === 'InsecureContextError'" class="message-error px-1">
                    [[ Camera access is only permitted in secure context. Use HTTPS or localhost rather than HTTP. ]]
                </p>
                <p v-else class="message-error px-1">[[ Camera error. ]]</p>
```

Reply:
I'll add `message-error px-1` to the parent container instead.

Reply:
See: https://github.com/Lombiq/Orchard-Vue.js/pull/81/commits/a142ed8b1dc013ddc97d0fea754016112320bb32